src/release/breaking-changes/image-provider-load-buffer.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Adding ImageProvider.loadBuffer
+description: ImageProviders should now be implemented using the new loadBuffer
+API instead of the existing load API.
+---
+
+## Summary
+
+* `ImageProvider` now has a method called `loadBuffer` which functions
```

Comment:
Consider linking some of API classes/functions mentioned here to the API docs for added context.